Boss/Mod/ChannelCreator/Manager.cpp

```diff
@@ -26,8 +29,8 @@
 namespace {
 
 /* Minimum and maximum channel size.  */
-auto const min_amount = Ln::Amount::btc(0.005);
-auto const max_amount = Ln::Amount::sat(16777215);
+auto min_amount = Ln::Amount::btc(0.005);
```

Comment:
Preferably we should instead rename these constants to `default_min_amount` and `default_max_amount`, then add `Ln::Amount min_amount` and  `Ln::Amount max_amount` to `class Manager` (and make `custom_min_amount` local to the triggered functions, or even just use the `class Manager` variables directly).

As a general principle we should avoid *mutable* (i.e. non-`const`/`constexpr`) [globals](https://wiki.c2.com/?GlobalVariablesConsideredHarmful) (even if only file-local scope, if its lifetime technically exceeds the life `main`, it is global), even if we know they will be used in classes that in practice will be single instance.

Reply:
OK I think I've reworked this to use the `class Manager` variables directly, as that seemed cleanest.

There's no sanity checking currently, and it's probably possible to silly things like configure `max < min` or to set these to other values which do not make sense.